evm-contracts/src/v0.4/ConversionProxy.sol

```diff
@@ -0,0 +1,132 @@
+pragma solidity 0.4.24;
+
+import "./vendor/SignedSafeMath.sol";
+import "./interfaces/AggregatorInterface.sol";
+import "./vendor/Ownable.sol";
+
+/**
+ * @title The ConversionProxy contract for Solidity 4
+ * @notice This contract allows for the rate of one aggregator
+ * contract to be represented in the currency of another aggregator
+ * contract.
+ */
+contract ConversionProxy is AggregatorInterface, Ownable {
+  using SignedSafeMath for int256;
+
+  uint8 public decimals;
+  AggregatorInterface public from;
+  AggregatorInterface public to;
+
+  event AddressesUpdated(
+    uint8 decimals,
```

Comment:
The ordering for these arguments is a counter intuitive, `decimals` is next to the `from` address but affects the `to` address. Either flipping the order of `to`/`from` OR moving decimals to be last would clarify the intention a bit.

Reply:
Went with putting `decimals` last when logging.